posthog/celery.py

```diff
@@ -101,6 +101,7 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
     )  # every day at a random minute past midnight. Randomize to avoid overloading license.posthog.com
 
     materialize_columns_crontab = get_crontab(settings.MATERIALIZE_COLUMNS_SCHEDULE_CRON)
```

Comment:
hmm, this is an EE setting, right? How does this work?

Similar lines to the question on the previous PR around splitting celery tasks to ee and non-ee.

Reply:
the actual materialize column task is stubbed so it doesn't actually do anything. but yes, would make sense to fully move this 🤔